--- a/pygco.py
+++ b/pygco.py
@@ -10,8 +10,8 @@
 _PAIRWISE_FLOAT_PRECISION = 1000
 _SMOOTH_COST_PRECISION = 100
 
-_int_types = [np.int, np.intc, np.int32, np.int64, np.longlong]
-_float_types = [np.float, np.float32, np.float64, np.float128]
+_int_types = [int, np.intc, np.int32, np.int64, np.longlong]
+_float_types = [float, np.float32, np.float64, np.float128]
 
 _SMALL_CONSTANT = 1e-10
 
@@ -404,4 +404,3 @@
     costH = np.ones((height, width-1), dtype=unary_cost.dtype)
 
     return cut_grid_graph(unary_cost, pairwise_cost, costV, costH, n_iter, algorithm)
-
