--- a/core/corr.py
+++ b/core/corr.py
@@ -1,6 +1,6 @@
 import torch
 import torch.nn.functional as F
-from utils.utils import bilinear_sampler, coords_grid
+from lib.RAFT.core.utils.utils import bilinear_sampler, coords_grid
 
 try:
     import alt_cuda_corr
--- a/core/raft.py
+++ b/core/raft.py
@@ -6,7 +6,7 @@
 from update import BasicUpdateBlock, SmallUpdateBlock
 from extractor import BasicEncoder, SmallEncoder
 from corr import CorrBlock, AlternateCorrBlock
-from utils.utils import bilinear_sampler, coords_grid, upflow8
+from lib.RAFT.core.utils.utils import bilinear_sampler, coords_grid, upflow8
 
 try:
     autocast = torch.cuda.amp.autocast
